packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -63,6 +64,10 @@ function AnimalInventory({ isCompactSideMenu, history }: AnimalInventoryProps) {
   const { t } = useTranslation(['translation', 'animal', 'common', 'message']);
   const theme = useTheme();
   const isDesktop = useMediaQuery(theme.breakpoints.up('lg'));
+
+  const role = useSelector(userFarmSelector);
+  const isWorker = 'role_id' in role ? role.role_id === 3 : false;
```

Comment:
@Tbrid Thank you for working on this, it works great!
Since we have `isAdminSelector` (in `userFarmSlice`), could you please replace the `isWorker` logic with: `const isAdmin = useSelector(isAdminSelector);`? 🙏
(`isAdmin` and `!isWorker` are equivalent here)